msteams-platform/concepts/device-capabilities/browser-device-permissions.md

```diff
@@ -31,7 +31,7 @@ A user will need to enable these permissions in the browser for these permission
 Teams app that requires device permissions in the browser must show instructions to users on where to find and enable these permissions in the Teams UI. Depending on the context in which your application is running, you need to ensure that your instructions are pointing the user to correct location to access these permissions. The permissions differ for personal apps, task module dialogs, tabs in chats, and channels or meetings.
 
 </br>
-<img src="../../assets/images/tabs/enable-access.png" alt="Enable camera access" width="800"/>
+:::image type="content" source="../../assets/images/tabs/enable-access.png" alt-text="The screenshot describes on how to enable camera.":::
```

Comment:
Read it out loud to understand if this makes sense "describes on how to...`
CC @v-habibaumme

Reply:
removed "described on how to" and added "shows the steps to"